tests/class-searchpress-unit-test-case.php

```diff
@@ -41,14 +52,76 @@ function search_and_get_field( $args, $field = 'post_name' ) {
 		return sp_results_pluck( $posts, $field );
 	}
 
-	protected function refresh_index() {
+	protected static function refresh_index() {
 		SP_API()->post( '_refresh' );
 	}
 
-	protected function index( $posts ) {
+	protected static function index( $posts ) {
 		$posts = is_array( $posts ) ? $posts : [ $posts ];
 		SP_API()->index_posts( $posts );
-		$this->refresh_index();
+		self::refresh_index();
+	}
+
+	protected static function create_sample_content() {
```

Comment:
I'd recommend adding docblocks to these functions, even though they're in a test class, to increase readability and comprehension of the test suite.